base/server/docs/installation/Installing_CA.md

```diff
@@ -6,6 +6,18 @@ Overview
 
 This page describes the process to install a CA subsystem with a self-signed CA signing certificate.
 
+Before beginning with the installation, please ensure that you have configured the directory
+server and added base entries. This step is described [here](http://www.dogtagpki.org/wiki/Installing_DS).
+
+Additionally, please verify that your FQDN is correctly reported by the following command:
+
+    python -c 'import socket; print(socket.getfqdn())'
```

Comment:
The doc is fine, but I'm afraid it has to be repeated in many other docs to be consistent. How about we execute this python code at the beginning of pkispawn to show the actual FQDN used during installation? We probably can also check, if the FQDN equals localhost or localhost.localdomain the install should fail.

Reply:
Perhaps we could have a "common practices" document and reference it from all of the other markdown files?

Reply:
We could, but if it gets to big people may not read everything. Regardless, I have no objection on this PR, but in general we should try to implement rules/guidelines as code whenever possible so that it will be executed automatically (i.e. make the product more robust).